README.md

```diff
@@ -57,6 +57,20 @@ The pipeline script consists of several steps:
 
 3. **Load**: Inserts the cleaned data into a Relational Database in this case we used PSQL. All data is inserted into the correct tables that can be viewed by running the bash scripts (See Bash Database Scripts Below)
 
+## ENV Variables
```

Comment:
Great addition, ensuring that anyone can understand what they need to run the project locally.